testapp/testapp.iml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$/.." external.system.id="GRADLE" external.system.module.group="webp-android-backport-library" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
```

Comment:
iml files should not be committed. We should use a gitignore (use the one of nichtlustig ;)